horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Test.java

```diff
@@ -28,9 +28,6 @@ public class Test {
     @Schema(implementation = String[].class)
     public JsonNode fingerprintLabels;
     public String fingerprintFilter;
-    @NotNull
-    @JsonProperty(required = true)
-    public Collection<View> views;
```

Comment:
Maybe we should take this separate

Reply:
View staying in -api caused a lot of changes to the tsx files. This actually made sense logically and also caused us to do less changes. win win :)